src/rules/converters/comment-format.ts

```diff
@@ -9,12 +9,13 @@ export const CapitalizedIgnoreMessage = "Only accepts a single string pattern to
 
 export const convertCommentFormat: RuleConverter = tslintRule => {
     const capitalizedRuleArguments: string[] = [];
-    const spaceCommentRuleArguments: string[] = [];
+    const spaceCommentRuleArguments: Array<string | { markers: string[] }> = [];
     const capitalizedNotices: string[] = [];
 
-    if (!tslintRule.ruleArguments.includes("check-space")) {
-        spaceCommentRuleArguments.push("never");
+    if (!spaceCommentRuleArguments.includes("always")) {
```

Comment:
Hmm, looks like you're missing unit test coverage for this line per the [test coverage report](https://circleci.com/gh/typescript-eslint/tslint-to-eslint-config/3039?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link). You'll need to add test coverage for this case to get it to pass.

Reply:
I get your point, will made another update

Reply:
But I do not know how. I tried to duplicate the test case on line 19, added another version without the `check-space` in `ruleArguments` array, but `npm run test:ci` still complain missing coverage on that line

https://github.com/typescript-eslint/tslint-to-eslint-config/pull/318/files/e51155daa587e4f8d6f54f145c85d76ab7c74ea4#diff-e32ee4f42ec93518baf8586b4fb733caL19

Reply:
The problem on line 15 is that `spaceCommentRuleArguments` is an empty array so the test suite is not satisfying the condition. Also, shouldn't it be `tslintRule.ruleArguments` as before?